README.md

```diff
@@ -29,6 +27,17 @@ composer require --dev dealerdirect/phpcodesniffer-composer-installer
 
 That's it.
 
+### Compatibility
+
+This plugin is compatible with:
+
+- PHP **5.x**, **7.x**, and **8.x** (Support for PHP v8 is available since [`v0.7.0`][v0.7])
```

Comment:
Just placing a note here for the record:
While we currently can only guarantee support with PHP 8.0, the current test build against PHP 8.1 _does_ seem to pass, so I'm fine with leaving this at `v8`.